views/summary.tpl.html

```diff
@@ -291,6 +291,20 @@ <h4 class="font-semibold text-lg text-gray-500">{{ .TotalTime | duration }}</h4>
                 </div>
             </div>
         </div>
+        
+        <div class="row-span-1 col-span-1 sm:row-span-3 sm:col-span-3 md:row-span-3 md:col-span-3 p-4 px-6 pb-10 bg-gray-850 text-gray-300 rounded-md shadow flex flex-col w-full no-break {{ if .IsProjectDetails }} hidden {{ end }}"
```

Comment:
Please add some margin to the card for consistency.

![image](https://github.com/user-attachments/assets/039ce9ac-8428-4af3-a9f1-c58a2681b492)